compiler/compiler.go

```diff
@@ -93,19 +92,15 @@ func New(pctx *proc.Context, inAST ast.Proc, adaptor proc.DataAdaptor, head *lak
 	if from != nil {
 		seq.Prepend(from)
 	}
-	entry, consts, err := semantic.Analyze(pctx.Context, seq, constsAST, adaptor, head)
+	entry, err := semantic.Analyze(pctx.Context, seq, adaptor, head)
 	if err != nil {
 		return nil, err
 	}
 	builder := kernel.NewBuilder(pctx, adaptor)
-	if err := builder.LoadConsts(consts); err != nil {
-		return nil, err
-	}
 	return &Runtime{
 		pctx:      pctx,
 		builder:   builder,
```

Comment:
Nit: There's no longer any need to define `builder` up above.
```suggestion
		builder:   kernel.NewBuilder(pctx, adaptor),
```